### PR TITLE
408 add computacenter namespace

### PR DESCRIPTION
**Pull request by aldavidson**
### Context

First part of [Trello card 408](https://trello.com/c/fsNGK22g/408-spin-up-first-api-end-point-to-share) - spin up a first draft API endpoint for Computacenter to notify us of orders placed

### Changes proposed in this pull request

* Add `/computacenter` namespaced `base_` and `home_` controllers
* Add distinguishing of computacenter users based on their email address
* Add basic placeholder computacenter homepage
* Restrict computacenter-namespaced controllers to computacenter users

### CODE REVIEW

File: app/views/computacenter/home/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      <%= t('page_titles.computacenter_home') %>
+    </h1>
+
+    <p class="govuk-body">Use this service to:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>download a CSV file of users who have permission to place orders on TechSource</li>
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--l">
```
Comment (fofr) [inline app/views/computacenter/home/show.html.erb:13]:
```suggestion
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -33,6 +33,7 @@ en:
     service_performance: Service performance
     accessibility: Accessibility statement
     support_responsible_bodies: On-boarded responsible bodies
+    computacenter_home: Increase children’s internet access
```
Comment (benilovj) [inline config/locales/en.yml:36]:
```suggestion
    computacenter_home: Users with permission to place orders and receive support through Computacenter
```

File: app/views/layouts/_nav.html.erb [hunk 3]
```diff
@@ -3,6 +3,8 @@
     <%= nav_item( title: "Service performance", url: support_service_performance_path )  %>
     <%= nav_item( title: "Responsible bodies", url: support_responsible_bodies_path )  %>
     <%= nav_item( title: "Background jobs", url: support_sidekiq_admin_path, html_options: {target: '_blank'} )  %>
+  <%- elsif @user&.is_computacenter? %>
+    <%= nav_item( title: 'Home', url: '#') %>
```
Comment (benilovj) [inline app/views/layouts/_nav.html.erb:7]: This is getting quite messy now. I wonder whether:

- `ApplicationController` should have an empty array of nav items
- `_nav.html.erb` should just show or display the nav items depending on whether the array is empty or not
- the Base controllers (or more specific controllers) are responsible for pre-populating the nav items per action?

Comment (aldavidson) [reply app/views/layouts/_nav.html.erb:7]: ....or we could just have a different partial for each kind of user?

I'll log a Tech Debt card on Trello to improve this

**Pull request merged**